src/Schema/PostgreSQLSchemaManager.php

```diff
@@ -632,11 +634,7 @@ protected function selectTableColumns(string $databaseName, ?string $tableName =
                 AND pg_index.indkey[0] = a.attnum
                 AND pg_index.indisprimary = 't'
             ) AS pri,
-            (SELECT pg_get_expr(adbin, adrelid)
-             FROM pg_attrdef
-             WHERE c.oid = pg_attrdef.adrelid
-                AND pg_attrdef.adnum=a.attnum
-            ) AS default,
+            ($fragmentDefaultValueSQL) AS default,
```

Comment:
Why does Codecov say hat line is not covered? Are you sure the new test is executed on the CI?

Reply:
let me check in the history of CI pipeline but I do remember to see one failed because a previous version had a bug

Reply:
https://github.com/doctrine/dbal/runs/22902499257 say 100% of my patch is test covered (if i open the details I got a blank page on firefox and chrome)  ?

Reply:
After that last push, it no longer complains, so I think this must be fine.

Reply:
does it mean we're good for merging  ? :)